go/registry/api/api.go

```diff
@@ -270,20 +273,44 @@ func NewRegisterRuntimeTx(nonce uint64, fee *transaction.Fee, sigRt *SignedRunti
 // EntityEvent is the event that is returned via WatchEntities to signify
 // entity registration changes and updates.
 type EntityEvent struct {
-	Entity         *entity.Entity
-	IsRegistration bool
+	Entity         *entity.Entity `json:"entity"`
+	IsRegistration bool           `json:"is_registration"`
 }
 
 // NodeEvent is the event that is returned via WatchNodes to signify node
 // registration changes and updates.
 type NodeEvent struct {
-	Node           *node.Node
-	IsRegistration bool
+	Node           *node.Node `json:"node"`
+	IsRegistration bool       `json:"is_registration"`
+}
+
+// RuntimeEvent signifies new runtime registration.
+type RuntimeEvent struct {
+	Runtime *Runtime `json:"runtime"`
+}
+
+// NodesExpiredEvent signifies node expirations.
+type NodesExpiredEvent struct {
+	Nodes []*node.Node `json:"nodes"`
+}
+
+// NodeUnfrozenEvent signifies when node becomes unfrozen.
+type NodeUnfrozenEvent struct {
+	NodeID signature.PublicKey `json:"node_id"`
+}
+
+// Event is a registry event returned via GetEvents.
+type Event struct {
+	RuntimeEvent      *RuntimeEvent      `json:"runtime_event,omitempty"`
```

Comment:
The serialized fields could omit the `_event` suffix (as the whole struct contains events).